### PR TITLE
bump k8s packages to 1.17.6

### PR DESCRIPTION
**Pull request by dprotaso**
Upstream change knative/pkg#1285

* Bumps K8s dependencies to 1.17.5
* This makes our minimum K8s version 1.16

### CODE REVIEW

File: hack/update-k8s-deps.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "k8s.io/apimachinery"
+  "k8s.io/client-go"
+  "k8s.io/code-generator"
+)
+
+function update_module {
+  local dep="${1}"
+  local version="${2}"
+
+
```
Comment (chaodaiG) [inline hack/update-k8s-deps.sh:36]: nit, empty line

Comment (dprotaso) [reply hack/update-k8s-deps.sh:36]: noooo.... i've rebased this so many times ;)

**Pull request merged**